existing-solutions.tex

```diff
@@ -1,74 +1,71 @@
 transactions to influence the result of the upcoming election.
 
-Next issue is the "long–range attack"\cite{pos_flaws_long}.
-In the very beginning, the stake is scattered among a small
-group of delegates that together have full control over the chain. After
-some time, they can cooperate and start a concurrent chain diverging from
-the main one. This could lead to nasty frauds and would
-destabilize trust over the chain.
+Next issue is the "long–range attack"\cite{pos_flaws_long}. In the very
+beginning, the stake is scattered among a small group of delegates that together
+have full control over the chain. After some time, they can cooperate and start
+a concurrent chain diverging from the main one. This could lead to nasty frauds
+and would destabilize trust over the chain.
 
-On the other hand, there are multiple approaches to deal with these problems. For instance,
-to prevent nothing–at–stake, the CASPER protocol introduces a "wrong voting
-penalty," which punishes
-voters who support conflicting forks\cite{casper}.
-However, this solution is backed by a finality gadget, which is located on another blockchain
-anyway. NXT deals with long–range attack by forcefully
-finalizing all blocks that are older than 720 generations\cite{nxt}.
-One must note that this doesn't actually solve the problem, but rather defers
-it. Moreover, it introduces weak subjectivity since one still needs to trust
-some entity while entering the network for the first time or after a longer
-downtime. Ouroboros staking system has managed to reach solid security, but
-at the cost of very high complexity\cite{ouroboros}.
+On the other hand, there are multiple approaches to deal with these problems.
+For instance, to prevent nothing–at–stake, the CASPER protocol introduces a
+"wrong voting penalty," which punishes voters who support conflicting
+forks\cite{casper}. However, this solution is backed by a finality gadget, which
+is located on another blockchain anyway. NXT deals with long–range attack by
+forcefully finalizing all blocks that are older than 720 generations\cite{nxt}.
+We believe that this doesn not actually solve the problem, but rather defers it.
+Moreover, it introduces weak subjectivity since one still needs to trust some
+entity while entering the network for the first time or after a longer downtime.
+Ouroboros staking system has managed to reach solid security, but at the cost of
+very high complexity\cite{ouroboros}.
 
-These are only a few examples. The general point is that PoS comes with many
-problems, which, as they are being solved, eventually introduce some new ones. This
-makes most of the pure PoS networks unreliable, and especially not as reliable
-as mature PoWs.
+Although these are only few examples, the general argument is that PoS comes
+with many problems, which, as they are being solved, eventually introduce new
+ones. This undermines reliability of PoS systems, and especially compared to
+mature PoWs.
```

Comment:
Of course, HC as proposed is still a PoS system, albeit with some techniques for addressing _some_ of the weaknesses of PoS.

Reply:
It is a hybrid: PoS with a PoW component, so not entirely PoS

Reply:
Changed wording